### PR TITLE
chore(lib): remove deprecated constructs

### PR DESCRIPTION
**Pull request by krokoko**
Fixes #

Remove deprecated constructs

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.

### CODE REVIEW

**Review by MichaelWalker-git — COMMENTED** [review]
LGTM Thanks for doing that. Sad about Kendra :(

**Review by MichaelWalker-git — APPROVED** [review]

**Comment by krokoko** [comment]
> LGTM Thanks for doing that. Sad about Kendra :(

Yes :( Removing since there was no adoption and it facilitates maintenance. We can revisit to see how to build a smaller construct with Kendra if there is a need

File: src/common/helpers/vpc-helper.ts [hunk 1]
```diff
@@ -211,6 +219,37 @@ export function suppressEncryptedLogWarnings(flowLog: FlowLog) {
   ]);
 }
 
+export function buildSecurityGroup(
```
Comment (dineshSajwan) [inline src/common/helpers/vpc-helper.ts:222]: Is this change related to this PR of removing deprecated construct? Looks like it's adding a new feature.

Comment (krokoko) [reply src/common/helpers/vpc-helper.ts:222]: Yes it is related to this PR. This function was located in a different helper file:

```import { buildSecurityGroup } from './kendra-helper';```

So moving it into the vpc helper file

**Review by dineshSajwan — APPROVED** [review]

**Pull request merged**